server/config/index.js

```diff
@@ -32,6 +34,19 @@ function anyEnabled() {
 	} );
 }
 
+const ssrConfig = template(
+	'var data = <%= data %>;' +
+	'<%= config %>' +
+	'<%= isEnabled %>' +
+	'<%= anyEnabled %>'
+) ( {
+	data: JSON.stringify( clientData ),
+	config: config.toString(),
```

Comment:
I just tested this out locally.  While I'd rather rely on the same module in both client/server, I couldn't figure out how to ensure webpack would exclude the serverConfig from the bundle (without changing what this file exports)